packages/constants/src/lib/constants/constants.ts

```diff
@@ -466,22 +466,50 @@ export const LIT_CHAINS: LITChain<LITEVMChain> = {
     type: null,
     vmType: 'EVM',
   },
+
+  /**
+   * Chainlist entry for the Chronicle Testnet.
+   * https://chainlist.org/chain/175177
+   */
   chronicleTestnet: {
     contractAddress: null,
     chainId: 175177,
     name: 'Chronicle - Lit Protocol Testnet',
-    symbol: 'testLPX',
+    symbol: 'tstLPX',
```

Comment:
I think this one is still off?
![image](https://github.com/user-attachments/assets/96ad68e0-e4b3-41d0-9f40-6bab9fec24fe)
```suggestion
    symbol: 'tstLIT',
```